docs/gambit/mutation-verifier.md

```diff
@@ -112,9 +110,16 @@ We support `"staging" : true` as an alternative to `"staging" : "master"`.
 Omitting this key will cause verification to run locally
   (unless the run script has it).
 - `"cloud"` : if you instead want to run on the cloud environment you can provide the `--cloud` flag. You can also add the name of a specific branch.
```

Comment:
- `"cloud"` : if you instead want to run it on the cloud environment you can provide the `--cloud` flag. You can also add the name of a specific branch.